.git-blame-ignore-revs

```diff
@@ -0,0 +1,2 @@
+# Format repository using black
+ee4dc48b7f9f5ce54fab47a3c3d9b3dea12c52f1
```

Comment:
?

Reply:
I covered that a bit in the PR description. I'll also elaborate on it more when I update our doc but this should explain: https://github.com/psf/black#migrating-your-code-style-without-ruining-git-blame

Reply:
you have to take the blame right? :-)